src/routes/tokens/ApiTokens.tsx

```diff
@@ -0,0 +1,299 @@
+import { useState, useEffect } from 'react';
+
+import {
+    Button,
+    Text,
+    VStack,
+    HStack,
+    Link,
+    Center,
+    Input,
+    Box,
+    Heading,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    useDisclosure,
+    Select,
+    Alert,
+    AlertIcon,
+    AlertTitle,
+    AlertDescription,
+    Tooltip,
+} from '@chakra-ui/react';
+import { RiAddLine, RiInformationLine } from 'react-icons/ri';
+
+import { listActiveTokens, requestToken } from '@/api/BearerTokenApi';
+import CopyIconButton from '@/features/copy/CopyIconButton';
+import { useToast } from '@/hooks/useToast';
+import { shortFormatDate, increaseDate } from '@/libs/utils';
+import ApiToken from '@/routes/tokens/components/ApiToken';
+import { BearerTokenT, NewBearerTokenT } from '@/types/BearerTokenTypes';
+
+import { parseToken, parseNewToken } from './BearerTokenUtils';
+
+const ApiTokens = () => {
+    const [activeTokens, setActiveTokens] = useState<BearerTokenT[]>([]);
+    const [generatedToken, setToken] = useState<NewBearerTokenT>();
+    const toast = useToast();
+
+    const [formNote, setFormNote] = useState('');
+    const [formExpiry, setFormExpiry] = useState('');
+    const [selectValue, setSelect] = useState('');
```

Comment:
What does `selectValue` correspond to?

Reply:
this is the value of the Select component